src/CoreBundle/private/library/classes/TCMSFields/TCMSFieldLookupDirectory.class.php

```diff
@@ -116,7 +116,7 @@ public function GetReadOnly()
     {
         $this->GetOptions();
         if (array_key_exists($this->data, $this->options)) {
-            return $this->_GetHiddenField().TGlobal::OutHTML($this->options[$this->data]);
```

Comment:
The OutHTML should be kept (also below).

Reply:
I don't know how it happened, but of course I added it again.